### ISSUE TITLE
Tips for training

### ISSUE THREAD

**Issue by leviethung2103** [twtygqyy/pytorch-SRResNet#15]
Hello, I would like to ask you about your experience in training the dataset. About choosing the iteration 500 (training will take about one week), size_average in MSE_Loss ? Why you set up size_average = False ? 
Thank you

**Comment by twtygqyy** [comment]
Hi, if you set size_average = True, then the loss would become very small, and when you use the loss for gradient calculation, the gradient will also be very small. No good for the convergence speed.
